q2cli/builtin/info.py

```diff
@@ -51,5 +61,31 @@ def info():
     click.secho('\nApplication config directory', fg='green')
     click.secho(q2cli.util.get_app_dir())
 
+    if config_level:
+        click.secho('\nConfig', fg='green')
+
+        config, action_executor_mapping, vendored_source = \
+            get_vendored_config()
+
+        click.secho(f'Config Source: {vendored_source}')
+
+        if action_executor_mapping:
+            config['parsl.executor_mapping'] = action_executor_mapping
+
+        if config_level > 1:
+            if config_level == 2:
+                config = dumps(config)
+            elif config_level == 3:
+                config['parsl'], _ = load_config_from_dict(config)
+
+            click.secho(f'\n{config}')
+
     click.secho('\nGetting help', fg='green')
     click.secho('To get help with QIIME 2, visit https://qiime2.org')
+
+    # When we have the final docs for this stuff up we should probably replace
+    # this with a reference to our docs which wil reference the parsl docs
```

Comment:
```suggestion
    # this with a reference to our docs which will reference the parsl docs
```

Reply:
Agreed.

Reply:
We could alternatively reference this for now, and get the relevant references to the Parsl docs in there:
https://develop.qiime2.org/en/latest/framework/how-to-guides/parallel-configuration.html

That URL will probably change at some point though, which is my only concern.

Reply:
Yeah which could cause old versions of QIIME 2 to produce a URL that 404s. Those docs already link to the parsl docs in multiple places, but the first link there 404s...

Reply:
@ebolyen what's your opinion?

Reply:
@Oddant1, let's discuss strategy for this in the engineering meeting today.